crates/common/src/lib.rs

```diff
@@ -426,47 +426,49 @@ impl std::fmt::Display for Chain {
     }
 }
 
-#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize, Dummy)]
-pub struct StarknetVersion(String);
+#[derive(Clone, Debug, Default, PartialEq, Eq, PartialOrd, Ord, Dummy)]
+pub struct StarknetVersion(u8, u8, u8, u8);
```

Comment:
Ah cool I was wondering if `Ord` would just work for tuples like this. Makes sense that it would.